docs/PFC-test-plan.md

```diff
@@ -251,3 +255,26 @@ Verify Device Under Test (DUT) processes the PFC PAUSE frame with lossy prioriti
    * Keysight Rx port should receive all the 'Background data traffic' as well as 'Test data traffic'. There should not be any loss observed.
 5. Stop the PFC PAUSE storm.
 6. Repeat the test with a different lossless priority value (!=Pi).
+   
+### Test Case #3 - GLOBAL PAUSE (IEEE 802.3x port based Flow Control)
+
+<b>Note</b> : 802.3x Port based Flow Control is different than IEEE 802.1Qbb PFC (Priority Flow Control)
+#### Test Objective
+Verify Device Under Test (DUT) processes the GLOBAL PAUSE frame.
+#### Test Configuration
+
+- On SONiC Device Under Test (DUT) configure  lossless priority of all values Pi (0 \<= i \<=
+    7).
+- Configure following traffic items on the Keysight device:
+  1. Test data traffic: A traffic item from the Keysight Tx port with all the 8 priorities Pi (0<= i <= 7), DSCP values from (0 - 63) and enable flow group by PFC Queue. Traffic should be configured with 100% line rate.
```

Comment:
The description of test data traffic is a little bit unclear. You can change it to "A traffic item from the Keysight Tx port to Rx port. The traffic item uses all the 64 DSCP values (0-63). PFC is enabled at all the 8 priorities of Tx port. The traffic demand is 100% line rate."

Reply:
Thank. Done